apps/telepathy/MyUserAgent.hxx

```diff
@@ -38,7 +38,7 @@ class MyUserAgent : public QObject, public recon::UserAgent, public resip::Threa
    Q_OBJECT
 public:
    MyUserAgent(recon::ConversationManager* conversationManager, std::shared_ptr<recon::UserAgentMasterProfile> profile, Connection& connection, std::shared_ptr<MyInstantMessage> instantMessage);
-   virtual void onApplicationTimer(unsigned int id, unsigned int durationMs, unsigned int seq);
+   void onApplicationTimer(unsigned int id, std::chrono::duration<double> duration, unsigned int seq) override;
```

Comment:
Nice find!  It would be great to convert all these overrides to using the override keyword to avoid this issue in the future with other overrides.